requirements.txt

```diff
@@ -1,4 +1,4 @@
-boto==2.38.0
+boto==2.39.0
```

Comment:
Release notes for boto 2.39.0: http://docs.pythonboto.org/en/latest/releasenotes/v2.39.0.html

Seems like a safe upgrade